internal/ethapi/api.go

```diff
@@ -907,6 +907,7 @@ func newRPCBalance(balance *big.Int) **hexutil.Big {
 	return &rpcBalance
 }
 
+// TODO: rename this API to EstimateL1MsgFee?
```

Comment:
yes, agree that changing to `EstimateL1MsgFee` is better.

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/351/commits/b34ec91bc5f4806c3c774f766f20905081d8f036